server/api/embedded_form.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+type EmbeddedForm struct {
```

Comment:
For (my) clarity - an EmbeddedForm (or another name) is still a Binding (with a implicit location associated)?  Do we consider Binding as the root/base type and EmbeddedForm as a specialization of it?

Reply:
For the time being, it is not a binding nor a form. It could be a binding, but then we would have to extend the properties a binding to allow all needed by a embed form. We could go back to my original idea (serveral months old) (`type Binding struct {AppID: string; Location: string; Call: AppCall; Props: interface{}}; Bindings: AppBindings[]`) and then define some utility types like `type PostEmbedProps struct {Title: string; Text: string;}` or `type ChannelHeaderProps struct {Label: string; IconURL: string}`.